arduinoCode/gyroTestingWMotor/gyroTestingWMotor.ino

```diff
@@ -1,30 +1,115 @@
-// MPU-6050 Short Example Sketch
-// By Arduino User JohnChi
-// August 17, 2014
-// Public Domain
+//Drive Straight w/Gyro (MPU-6050)
 #include<Wire.h>
+#include <Adafruit_MotorShield.h>
+
 const int MPU_addr=0x68;  // I2C address of the MPU-6050
-int16_t AcX,AcY,AcZ,Tmp,GyX,GyY,GyZ;
+int16_t GyZ;
+float elapsedTime, currentTime, previousTime,angle = 0, target_angle,offset = 0;
+
+//target speed to go straight.
+int target_speed = 70;
+
+Adafruit_MotorShield AFMS = Adafruit_MotorShield(0x61); // our shield's address
```

Comment:
You will need to change to shield address that you are using. No solder is address 0x60